projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -374,20 +352,18 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
 
     /** @hidden @internal */
     public get appliedFormat(): string {
-        if (this.formatter) {
-            return this.formatter(this.value);
-        }
-        if (!this.hasProjectedInputs) {
-            if (this.placeholder !== '') {
-                return this.placeholder;
-            }
-            // TODO: use displayFormat - see how shortDate, longDate can be defined
-            return this.inputFormat
-                ? `${this.inputFormat} - ${this.inputFormat}`
-                : `${DatePickerUtil.DEFAULT_INPUT_FORMAT} - ${DatePickerUtil.DEFAULT_INPUT_FORMAT}`;
-        } else {
-            return this.inputFormat || DatePickerUtil.DEFAULT_INPUT_FORMAT;
+        return DatePickerUtil.getLocaleDateFormat(this.locale, this.displayFormat)
+            || DatePickerUtil.DEFAULT_INPUT_FORMAT;
+    }
+
+    /** @hidden @internal */
+    public get singleInputFormat(): string {
+        if (this.placeholder !== '') {
+            return this.placeholder;
         }
+
+        const format = this.appliedFormat;
+        return `${format} - ${format}`;
```

Comment:
replace the dash (` - `) with constant, e.g.:
```
const SingleInputDatesConcatenationString = ' - ';
```
This will allow us later to easily expose some way to change this string.